lib/nacre/abstract_resource_collection.rb

```diff
@@ -48,5 +49,22 @@ def self.extract_resources(json)
     def self.json_to_params(json_order)
       format_hash_keys(json_order)
     end
+
+    def self.service_url
+      "#{configuration.resource_url}/#{service_name}-service"
+    end
+
+    def self.url
+      service_url + '/' + resource_name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.